src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java

```diff
@@ -127,24 +112,33 @@ public void setVaultAccessor(VaultAccessor vaultAccessor) {
         return leaseIds;
     }
 
-    private List<LogicalResponse> provideEnvironmentVariablesFromVault(Context context, Run build) throws VaultException {
+    private List<LogicalResponse> provideEnvironmentVariablesFromVault(Context context, Run build) {
         String url = getConfiguration().getVaultUrl();
 
         if (StringUtils.isBlank(url)) {
             throw new VaultPluginException("The vault url was not configured - please specify the vault url to use.");
         }
 
         VaultCredential credential = retrieveVaultCredentials(build);
-
-        vaultAccessor.init(url, credential);
+      
+        vaultAccessor.init(url, credential, configuration.isSkipSslVerification());
         ArrayList<LogicalResponse> responses = new ArrayList<>();
         for (VaultSecret vaultSecret : vaultSecrets) {
-            LogicalResponse response = vaultAccessor.read(vaultSecret.getPath());
-            responses.add(response);
-            Map<String, String> values = response.getData();
-            for (VaultSecretValue value : vaultSecret.getSecretValues()) {
-                valuesToMask.add(values.get(value.getVaultKey()));
-                context.env(value.getEnvVar(), values.get(value.getVaultKey()));
+            vaultAccessor.auth(credential);
```

Comment:
```
[ERROR] /Users/ptierno/Projects/java/hashicorp-vault-plugin/src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java:[127,26] cannot find symbol
  symbol:   method auth(com.datapipe.jenkins.vault.credentials.VaultCredential)
  location: variable vaultAccessor of type com.datapipe.jenkins.vault.VaultAccessor
```

The credential performs the auth, not the accessor.

https://github.com/jenkinsci/hashicorp-vault-plugin/blob/master/src/main/java/com/datapipe/jenkins/vault/credentials/VaultCredential.java#L15

Reply:
confused 😕 That line comes from here https://github.com/jenkinsci/hashicorp-vault-plugin/blob/3dda0cabd05c3cb8cad45ce85f7ab7b16a69aa86/src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java#L131
I didn't mean to change the logic...
I try to fix that again...